2018-09-28 19:11:51.852 32335 DEBUG oslo_db.sqlalchemy.engines [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-28 19:11:56.550 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 19:11:56.557 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 19:11:56.580 32335 DEBUG oslo_db.sqlalchemy.engines [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-28 19:11:56.838 32335 DEBUG nova.context [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),7d2d29ce-c1fb-484d-a8c0-d22d93f3a902(default) load_cells /usr/lib/python2.7/site-packages/nova/context.py:508
2018-09-28 19:11:56.840 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 19:11:56.843 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 19:11:56.845 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 19:11:56.846 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 19:11:56.901 32335 DEBUG oslo_db.sqlalchemy.engines [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-28 19:11:56.950 32335 DEBUG nova.quota [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Getting quotas for project 4db06b0f9b2448bdb7fd5031b1ecc2fd. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-09-28 19:11:56.968 32335 DEBUG nova.quota [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Getting quotas for user a7dbac30f0564a7cb747ccee9e7861c0 and project 4db06b0f9b2448bdb7fd5031b1ecc2fd. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-09-28 19:11:57.007 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 19:11:57.009 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 19:11:57.010 32335 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:11:57.087 32335 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:11:57.104 32335 DEBUG nova.objects.instance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 19:11:57.178 32335 DEBUG nova.conductor.manager [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] [instance: 0d1efc28-be52-463b-8b21-cabbc3319b2f] block_device_mapping [BlockDeviceMapping(attachment_id=<?>,boot_index=0,connection_info=None,created_at=<?>,delete_on_termination=True,deleted=<?>,deleted_at=<?>,destination_type='local',device_name=None,device_type='disk',disk_bus=None,guest_format=None,id=<?>,image_id='ce719f98-f39f-4fd9-b9c0-95af5960d9f2',instance=<?>,instance_uuid=<?>,no_device=False,snapshot_id=None,source_type='image',tag=None,updated_at=<?>,volume_id=None,volume_size=None)] _create_block_device_mapping /usr/lib/python2.7/site-packages/nova/conductor/manager.py:968
2018-09-28 19:11:57.184 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 19:11:57.186 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 19:11:57.311 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-28 19:11:57.312 32335 DEBUG oslo_concurrency.lockutils [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lock "7d2d29ce-c1fb-484d-a8c0-d22d93f3a902" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-28 19:11:57.829 32334 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:11:57.925 32334 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:11:58.471 32342 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:11:58.545 32342 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:12:00.138 32337 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:12:00.212 32337 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:12:00.640 32338 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:12:00.728 32338 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:12:01.014 32342 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:12:01.038 32342 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:12:15.668 32335 WARNING nova.image.glance [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:12:15.695 32335 DEBUG nova.notifications.objects.base [req-e31faace-d0e1-4668-b81d-e404b1836458 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:14:49.315 32338 WARNING nova.image.glance [req-d742774b-aa83-43cc-957d-f24bbba223e0 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:14:49.341 32338 DEBUG nova.notifications.objects.base [req-d742774b-aa83-43cc-957d-f24bbba223e0 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:14:49.346 32338 DEBUG nova.objects.instance [req-d742774b-aa83-43cc-957d-f24bbba223e0 a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 19:15:22.632 32334 WARNING nova.image.glance [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:15:22.662 32334 DEBUG nova.notifications.objects.base [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:15:22.668 32334 DEBUG nova.objects.instance [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 19:15:23.100 32339 WARNING nova.image.glance [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:15:23.192 32339 DEBUG nova.notifications.objects.base [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:15:23.201 32339 DEBUG nova.objects.instance [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 19:15:30.795 32335 WARNING nova.image.glance [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:15:30.827 32335 DEBUG nova.notifications.objects.base [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:15:30.833 32335 DEBUG nova.objects.instance [req-cdf8cff8-dab7-49e2-a530-b00d4815902f a7dbac30f0564a7cb747ccee9e7861c0 4db06b0f9b2448bdb7fd5031b1ecc2fd - c3012869344c4ca19b8a3abadaff543c c3012869344c4ca19b8a3abadaff543c] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 19:17:14.475 32335 WARNING nova.image.glance [req-97626307-ff5f-4585-98df-3de27b78500a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:17:14.503 32335 DEBUG nova.notifications.objects.base [req-97626307-ff5f-4585-98df-3de27b78500a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:17:14.507 32335 DEBUG nova.objects.instance [req-97626307-ff5f-4585-98df-3de27b78500a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-09-28 19:17:31.058 32341 WARNING nova.image.glance [req-97626307-ff5f-4585-98df-3de27b78500a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-09-28 19:17:31.162 32341 DEBUG nova.notifications.objects.base [req-97626307-ff5f-4585-98df-3de27b78500a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Defaulting the value of the field 'projects' to None in FlavorPayload due to 'Cannot call _load_projects on orphaned Flavor object' populate_schema /usr/lib/python2.7/site-packages/nova/notifications/objects/base.py:125
2018-09-28 19:17:31.170 32341 DEBUG nova.objects.instance [req-97626307-ff5f-4585-98df-3de27b78500a c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - default default] Lazy-loading 'tags' on Instance uuid 0d1efc28-be52-463b-8b21-cabbc3319b2f obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
